crates/re_viewer_context/src/time_control.rs

```diff
@@ -112,11 +114,17 @@ impl Default for TimeControl {
 
 impl TimeControl {
     /// Update the current time
-    pub fn move_time(&mut self, times_per_timeline: &TimesPerTimeline, stable_dt: f32) {
+    #[must_use]
+    pub fn move_time(
```

Comment:
nit: we've gone pretty far past "moving time" at this point... this is more of `TimeControl::update_state()`

Reply:
The bulk of what it does is move the time though. The only other thing it does is make sure we are on a valid timeline

Reply:
And changes the state of the playback controls.